task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -10,18 +10,30 @@ static void Main(string[] args)
             int firstNumber = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the second number");
             int lastNumber = int.Parse(Console.ReadLine());
+            int min = 0;
+            int max = 0;
 
-            for (int i = firstNumber; i <= lastNumber; i++)
+            if (firstNumber < lastNumber || firstNumber == lastNumber)
+            {
+                min = firstNumber;
+                max = lastNumber;
+            }
+            else if (firstNumber > lastNumber)
```

Comment:
Why do you explicitly check that first is more then second? I guess else would be enough here.